packages/generators/init-generator.ts

```diff
@@ -158,6 +168,7 @@ export default class InitGenerator extends Generator {
 			)
 			.then(
 				(stylingTypeAnswer: { stylingType: string }): void => {
+					this.configuration.answers.push(stylingTypeAnswer);
```

Comment:
No need to push to create more objects on heap. Check directly against `this.configuration`

Reply:
Cool, on it